src/index.html

```diff
@@ -7,6 +7,312 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div id="home">
+        <a href="#home" class="logo">
+          <img src="./images/logo.png" alt="Logo">
+        </a>
+      </div>
+      <nav class="nav">
```

Comment:
```suggestion
      </div>
      
      <nav class="nav">
```

Adding a space here makes the code easier to read.

Reply:
Yep, I did this one before figuring out those rules. Correcting..